### ISSUE TITLE
ReferenceError: Can't find variable: File

### ISSUE THREAD

**Issue by jmmoser** [oven-sh/bun#4229]
### What version of Bun is running?

0.7.3

### What platform is your computer?

Darwin 22.6.0 arm64 arm

### What steps can reproduce the bug?

`echo "console.log(File);" > test.ts && bun run test.ts`

### What is the expected behavior?

Doesn't throw an error and logs something like `[class File]`. Node logs this: `[class File extends Blob]`

### What do you see instead?

```
1 | console.log(File);
               ^
ReferenceError: Can't find variable: File
```

### Additional information

Detected after upgrading openai dependency to v4.

**Comment by Jarred-Sumner** [comment]
The `File` constructor is not implemented yet.

**Comment by Jarred-Sumner** [comment]
Fixed in c99a9ba33a36f16f79e38ff77423f5d301f6d2a8
